src/game/Strategic/Game_Clock.cc

```diff
@@ -961,3 +924,14 @@ void ClearTacticalStuffDueToTimeCompression( void )
 		CencelAllActionsForTimeCompression( );
 	}
 }
+
+
+void UpdateGameClockGlobals()
+{
+	//Calculate the day, hour, and minutes.
+	guiDay = guiGameClock / NUM_SEC_IN_DAY;
+	guiHour = (guiGameClock - guiDay * NUM_SEC_IN_DAY) / NUM_SEC_IN_HOUR;
+	guiMin = (guiGameClock - (guiDay * NUM_SEC_IN_DAY +  guiHour * NUM_SEC_IN_HOUR)) / NUM_SEC_IN_MIN;
+
+	WORLDTIMESTR = ST::format("{} {}, {02d}:{02d}", gpGameClockString, guiDay, guiHour, guiMin);
```

Comment:
Some of the uses were of `pDayStrings`, not `gpGameClockString` — is this really ok?

Reply:
In all translation except the Italian these strings are identical:

```
_ChineseText.cc:static const ST::string s_cn_gpGameClockString = "日";
_ChineseText.cc:static const ST::string s_cn_pDayStrings = "日";
_DutchText.cc:static const ST::string s_dut_gpGameClockString = "Dag";
_DutchText.cc:static const ST::string s_dut_pDayStrings = "Dag";
_EnglishText.cc:static const ST::string s_eng_gpGameClockString = "Day";
_EnglishText.cc:static const ST::string s_eng_pDayStrings = "Day";
_FrenchText.cc:static const ST::string s_fr_gpGameClockString = "Jour";
_FrenchText.cc:static const ST::string s_fr_pDayStrings = "Jour";
_GermanText.cc:static const ST::string s_ger_gpGameClockString = "Tag";
_GermanText.cc:static const ST::string s_ger_pDayStrings = "Tag";
_ItalianText.cc:static const ST::string s_it_gpGameClockString = "Gg";
_ItalianText.cc:static const ST::string s_it_pDayStrings = "Giorno";
_PolishText.cc:static const ST::string s_pl_gpGameClockString = "Dzień";
_PolishText.cc:static const ST::string s_pl_pDayStrings = "Dzień";
_RussianText.cc:static const ST::string s_rus_gpGameClockString = "День";
_RussianText.cc:static const ST::string s_rus_pDayStrings = "День";
```
Looks more like a bug in the Italian version to me, it would be weird if the way the clock is displayed depended on which function was called last.

Reply:
No, it could be a maximum length thing — there's limited space available in the "widget".